libraries/stdlib/src/kotlin/properties/Delegates.kt

```diff
@@ -65,5 +65,7 @@ private class NotNullVar<T : Any>() : ReadWriteProperty<Any?, T> {
     public override fun setValue(thisRef: Any?, property: KProperty<*>, value: T) {
         this.value = value
     }
+
+    public override fun toString(): String = if (value != null) value.toString() else "Not-null value not initialized yet."
```

Comment:
I've experimented with toString variants in the debugger, and I think that for this purpose it would be better to keep things consistent with other delegates provided by `Delegates` object.
```suggestion
    public override fun toString(): String = 
        "NotNullProperty(${if (value != null) "value=$value" else "value not initialized yet"})"
```